### PR TITLE
Additional text fixtures

### PR DESCRIPTION
**Pull request by arthurian**
This PR adds text fixtures for latin  and russian.

- The latin text (`livy.json`) is from http://www.thelatinlibrary.com/livy/liv.1.shtml
- The russian text (`tolstoy.json`) is from https://ilibrary.ru/text/1099/p.1/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ pipenv run pip install pip==18.0
 pipenv install
 pipenv shell
 ./manage.py migrate
-./manage.py loaddata sites cana
+./manage.py loaddata sites cana livy tolsoy
```
Comment (jtauber) [inline README.md]:
```suggestion
./manage.py loaddata sites cana livy tolstoy
```
I may have typo'd that when I resolved the merge conflict

Comment (jtauber) [reply README.md]: no, it already there before :-)

**Comment by jtauber** [comment]
Good to go once the README typo is fixed.

File: README.md (hunk 1 above)
Comment (arthurian) [reply README.md]: Good catch @jtauber. Fixed.

**Pull request merged**